Raspberry_Pi_Scheduled_Scripts/get_and_set_rankings.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
Since this will be a ```cron job```, please remove ```print``` statements for both of the scripts. Logging is a good idea, each print can be logged, along with time started running, finished, the time it took, the number of rows successfully updated, and the number failed (which ones failed).

 Please add how to run this code on docstring. Will ```python get_and_set_rankings.py``` for instance work?

Can we run both of these scripts every 10 minutes (@SoloNick cron ```*/10 * * * *```)?

Reply:
@MaksimEkin